src/helpers/useFocusItemHelper.ts

```diff
@@ -0,0 +1,171 @@
+import _ from 'lodash';
```

Comment:
Few comments on this.. 
- Is this hook really a common one or mainly for TabController purpose? 
I assume you want to use it in FilterChips as well, in that case, consider moving it to Hooks folder. 
- The name of the hook needs a little work in my opinion `..Helper` doesn't really say much, clearly it's a hook that helps doing something (: Let's try to find a better name.

Reply:
I'll move it to the hooks

useFocusItem
useScrollToItem

Reply:
I think useScrollIToItem hints better what the hook does..

Reply:
Done